examples/weathertabs/js/components/WeatherTab.jsx

```diff
@@ -0,0 +1,68 @@
+var React = require('react');
+var AltManagerMixin = require('../../../../mixins/AltManagerMixin');
+var AppActions = require('../actions/AppActions');
+var WeatherActions = require('../actions/WeatherActions');
+var WeatherStore = require('../stores/WeatherStore');
+
+module.exports = React.createClass({
+
+  displayName: 'WeatherApp',
+
+  mixins: [AltManagerMixin],
+  statics: { registerStore: WeatherStore, registerAction: WeatherActions },
+
+  onNewStore: function(state, newProps) {
```

Comment:
yeah I;'ll do that. the problem is when moving to a composite component this method won't be available.